test/unittests/execute_test.cpp

```diff
@@ -39,9 +39,9 @@ TEST(execute, select)
 {
     /* wat2wasm
     (func (param i64 i64 i32) (result i64)
-      get_local 0
-      get_local 1
-      get_local 2
+      local.get 0
```

Comment:
Both of these are valid, are we using this form now consistently?

Reply:
I'm using `local.xxx` consistently for some time :) Not all instructions have been converted to the newer form.

Reply:
There are still other occurrences of `{set,get}_local` in other files.

Reply:
But is this first commit making it uniform, or there are still places using `get_local`?

Reply:
There are other files having `get_local`.

Reply:
I always forget what is the preference.

Reply:
The ones with the `.`.

Reply:
> There are other files having get_local.

Is it too many to change them here and now?